packages/console/src/components/MultiTextInput/index.tsx

```diff
@@ -1,25 +1,33 @@
-import React, { useMemo } from 'react';
+import { I18nKey } from '@logto/phrases';
+import classNames from 'classnames';
+import React, { useMemo, useState } from 'react';
 import { useTranslation } from 'react-i18next';
+import Modal from 'react-modal';
 
 import * as textButtonStyles from '@/components/TextButton/index.module.scss';
 import Minus from '@/icons/Minus';
+import * as modalStyles from '@/scss/modal.module.scss';
 
 import IconButton from '../IconButton';
 import TextInput from '../TextInput';
+import DeleteConfirm from './DeleteConfirm';
 import * as styles from './index.module.scss';
 import { MultiTextInputError } from './types';
 
 type Props = {
+  title: I18nKey;
   value?: string[];
   onChange: (value: string[]) => void;
   error?: MultiTextInputError;
 };
 
-const MultiTextInput = ({ value, onChange, error }: Props) => {
+const MultiTextInput = ({ title, value, onChange, error }: Props) => {
   const { t } = useTranslation(undefined, {
     keyPrefix: 'admin_console',
   });
 
+  const [deleteFieldIndex, setDeleteFieldIndex] = useState<number | undefined>();
```

Comment:
nit: just `useState<number>()` would be good

Reply:
updated